docs/flatcar-container-linux/_index.md

```diff
@@ -4,17 +4,12 @@ title: Flatcar Container Linux
 children_are_versions: true
 external_docs:
   - repo: https://github.com/kinvolk/flatcar-docs.git
-    name: "2605.7.0"
+    name: "2605"
     branch: "main"
     dir: "docs"
 sidebar:
   skip: true
   link: latest
-cascade:
```

Comment:
Actually, we should keep them here (they are template variables) but have to remove them from `flatcar-docs`